packages/design-system-react/src/components/badge-count/BadgeCount.constants.ts

```diff
@@ -0,0 +1,23 @@
+import { TextVariant } from '../text';
+import { BadgeCountSize } from './BadgeCount.types';
+
+// Mappings
+export const MAP_BADGECOUNT_SIZE_TEXTVARIANT: Record<
+  BadgeCountSize,
+  TextVariant
+> = {
+  [BadgeCountSize.Md]: TextVariant.BodyXs,
+  [BadgeCountSize.Lg]: TextVariant.BodySm,
+};
+export const MAP_BADGECOUNT_SIZE_LINEHEIGHT: Record<BadgeCountSize, string> = {
+  [BadgeCountSize.Md]: 'leading-[14px]', // line-height 14px
+  [BadgeCountSize.Lg]: 'leading-4', // line-height 16px
+};
```

Comment:
Do we need line height? It looks like it is being overridden by the TextVariant anyway so this isn't changing anything.

<img width="1511" alt="Screenshot 2025-03-26 at 1 40 16 PM" src="https://github.com/user-attachments/assets/14427b51-ccf8-4b52-bb02-0bcaace8a7fd" />

Reply:
Removed here https://github.com/MetaMask/metamask-design-system/pull/511/commits/746126c5952f761bd2482b18cf6bbdcd295e7fc4